test/e2e/specs/specs-playwright/wp-likes__post-spec.js

```diff
@@ -66,26 +62,21 @@ describe( DataHelper.createSuiteTitle( 'Likes (Post)' ), function () {
 		it( 'Unlike post', async function () {
 			await publishedPostPage.unlikePost();
 		} );
+	} );
 
-		it( 'Clear cookies', async function () {
-			await BrowserManager.clearAuthenticationState( page );
+	describe( 'As the liking user', () => {
+		beforeAll( async () => {
+			const testAccount = new TestAccount( likeUser );
+			await testAccount.authenticate( page );
+		} );
+
+		it( 'Go to the published post page', async () => {
+			await page.goto( publishedURL );
 		} );
 
 		it( `Like post as ${ likeUser }`, async function () {
 			publishedPostPage = new PublishedPostPage( page );
-
-			// Clicking the Like button will bring up a new popup.
-			// Observe for the new popup in the Promise. Once the Page object
-			// is obtained, execute login steps there and wait for the
-			// popup to close.
-			await Promise.all( [
-				page.on( 'popup', async ( popup ) => {
-					const loginPage = new LoginPage( popup );
-					await loginPage.loginFromPopup( { account: likeUser } );
-					await popup.waitForEvent( 'close' );
-				} ),
-				publishedPostPage.likePost(),
-			] );
+			await publishedPostPage.likePost();
```

Comment:
Does this change preserve the steps of:
- user is logged out
- user clicks on Like Post button
- user is prompted to log in
- once logged in, Like is recorded

Reply:
Good point, I missed the fact that the popup itself might be a subject of the test here. Will update!

Reply:
I'll handle that in a follow-up PR where we can separate the context properly for both, posting and liking users: #59811.

Reply:
https://github.com/Automattic/wp-calypso/pull/59811/commits/281250c606b838029a870185cbc36f9374ef811d